components/Banner/examples/supplemental/MDCBannerParams.h

```diff
@@ -0,0 +1,51 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma mark - Layout Constants
+
+static const CGFloat kIconImageContainerSideLength = 40.0f;
+
+static const CGFloat kIconImageSideLength = 24.0f;
+
+static const CGFloat kLeadingPadding = 16.0f;
+
+static const CGFloat kTrailingPadding = 8.0f;
+
+static const CGFloat kTrailingPaddingLarge = 16.0f;
+
+static const CGFloat kTopPaddingSmall = 10.0f;
+
+static const CGFloat kTopPaddingLarge = 24.0f;
+
+static const CGFloat kBottomPadding = 8.0f;
+
+static const CGFloat kButtonHorizontalIntervalSpace = 8.0f;
+
+static const CGFloat kButtonVerticalIntervalSpace = 8.0f;
+
+static const CGFloat kSpaceBetweenIconImageAndTextLabel = 16.0f;
+
+static const CGFloat kHorizontalSpaceBetweenTextLabelAndButton = 36.0f;
+
+static const CGFloat kVerticalSpaceBetweenButtonAndTextLabelLarge = 20.0f;
+
+static const CGFloat kVerticalSpaceBetweenButtonAndTextLabelSmall = 12.0f;
+
+#pragma mark - Text Values
+
+static const CGFloat kTextColorOpacity = 0.87f;
+
+static const CGFloat kTextFontSize = 14.0f;
+
+static const NSInteger kTextNumberOfLineLimit = 3;
```

Comment:
Please fold all of this file's constants into their respective .m files.

Reply:
Addressed in 2b23608